### PR TITLE
Add GitHub Workflow Replacement for Jenkins job_PreCommit_Community_Metrics

### PR DESCRIPTION
**Pull request by aleksandr-dudko**
The following jenkins jobs have been migrated:

https://ci-beam.apache.org/view/PreCommit/job/beam_PreCommit_CommunityMetrics_Commit/
https://ci-beam.apache.org/view/PreCommit/job/beam_PreCommit_CommunityMetrics_Cron/
https://ci-beam.apache.org/view/PreCommit/job/beam_PreCommit_CommunityMetrics_Phrase/

How has it been tested?
The job has been tested on Self-hosted GKE runners ubuntu 20.04.
The following test cases are passed:

by creating pull requests with changes in trigger paths
by adding a comment to the PR
by making a push
Screenshot from 2023-07-10 17-46-23

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/get-started-contributing/#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

File: .github/workflows/job_PreCommit_CommunityMetrics.yml [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  issues: read
+  discussions: read
+  packages: read
+  pages: read
+  repository-projects: read
+  security-events: read
+  statuses: read
+
+jobs:
+  CommunityMetricsPrecommit:
```
Comment (andreydevyatkin) [inline .github/workflows/job_PreCommit_CommunityMetrics.yml:47]: We should use the same job name as Jenkins to keep it consistent. Please rename to `job_PreCommit_CommunityMetrics`

File: .github/workflows/job_PreCommit_CommunityMetrics.yml [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    name: Community Metrics
```
Comment (andreydevyatkin) [inline .github/workflows/job_PreCommit_CommunityMetrics.yml:48]: Rename to `job_PreCommit_CommunityMetrics`

**Review by andreydevyatkin — COMMENTED** [review]
see the inline comments

File: .github/workflows/job_PreCommit_CommunityMetrics.yml (hunk 1 above)
Comment (aleksandr-dudko) [reply .github/workflows/job_PreCommit_CommunityMetrics.yml]: It's done.

File: .github/workflows/job_PreCommit_CommunityMetrics.yml (hunk 2 above)
Comment (aleksandr-dudko) [reply .github/workflows/job_PreCommit_CommunityMetrics.yml]: It's done.

File: .test-infra/metrics/build.gradle [hunk 3]
```diff
@@ -27,7 +27,7 @@ ext {
   cluster = 'metrics'
   zone='us-central1-a'
   projectName='apache-beam-testing'
-  kubeConfigPath='/home/jenkins/.kube/config'
+  kubeConfigPath = project.properties['KUBE_CONFIG_PATH'] ?: '/home/jenkins/.kube/config'
```
Comment (Abacn) [inline .test-infra/metrics/build.gradle:30]: (just a note) may clean up this once completely migrate off jenkins (`'/home/jenkins/.kube/config'` -> `"/home/${System.getenv('ENVIRONMENTAL')}/.kube/config"`)

**Pull request merged**